clang/test/Driver/sycl-offload.cpp

```diff
@@ -0,0 +1,9 @@
+/// Check "-aux-target-cpu" and "target-cpu" are passed when compiling for SYCL offload device and host codes:
+//  RUN:  %clang -### -fsycl -c %s 2>&1 | FileCheck -check-prefix=CHECK-OFFLOAD %s
+//  CHECK-OFFLOAD: "-aux-target-cpu" "x86-64"
+//  CHECK-OFFLOAD: "-target-cpu" "x86-64"
+
+/// Check "-aux-target-cpu" and "-target-cpu" are passed when compiling for SYCL offload device and host codes:
```

Comment:
One more nit:  You are also checking for feature here, so adding that in the comment would reduce confusion from the first check which uses the same comment.

Reply:
Thanks @mdtoguchi. I have updated comments.